code/sorting/src/bubble_sort/bubble_sort.coffee

```diff
@@ -0,0 +1,12 @@
+Bubble_Sort = (list) ->
+  for i in [0...list.length]
+    for j in [0...list.length - i]
+      [list[j], list[j+1]] = [list[j + 1], list[j]] if list[j] > list[j + 1]
```

Comment:
Explanatory comment for line would be good. Also put spaces around the plus sign in `list[j+1]`.